src/govActions.sol

```diff
@@ -13,7 +13,7 @@ contract Setter {
     function init(bytes32) public;
 }
 
-contract Plan {
+contract GovActions {
```

Comment:
`Book`?

Reply:
I agree `Plan` is confusing, I think `Book` as well. They are actions for a proxy.
Not sure if there is a proper 4 letter meme for it.